### PR TITLE
Add html head

### PR DESCRIPTION
**Pull request by Lea-Mentz**

### CODE REVIEW

**Review by MathiSans — APPROVED** [review]
✅

**Review by YavorLalev — APPROVED** [review]
Aproved!

**Review by Ramin-faqyri — APPROVED** [review]
good structure dude

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body></body>
+</html>
```
Comment (YavorLalev) [inline html/index.html:9]: Where is the footer?

**Pull request merged**